features/base/test/test_debsums.py

```diff
@@ -12,5 +12,5 @@
     ]
 )
 
-def test_debsums(client, exclude):
+def test_debsums(client, exclude, chroot):
```

Comment:
Instead of pinning this only to `chroot`, we probably should create a new fixture (e.g. `non_fedramp` that only prevents it from running when `fedramp` feature is activated.

Reply:
no, this test should really only run on chroot. Since /usr is read-only and it will not be possible to install anything except in the chroot environment.

Reply:
Ok, I'm fine with it in this case (just struggled because of the branch name).

lgtm